src/intl/en/learn-quizzes.json

```
@@ -324,5 +324,41 @@
+  "k002-b-label": "Sends the proof to the user for storage",
+  "k002-b-explanation": "Users are not expected to hold onto the results of their transaction. This information is posted to Mainnet.",
+  "k002-c-label": "Submit the results to Ethereum",
+  "k002-c-explanation": "Layer 2 rollups post the results of their transaction execution to Mainnet, securing it in Ethereum's history",
+  "k002-d-label": "Delete the result to reduce costs",
+  "k002-d-explanation": "Layer 2 rollups post the results of their transaction execution to Mainnet. The cost savings achieved with this approach is by bundling and compressing transaction data, and ultimately storing it in cheap storage that expires once made available to those who need it.",
+  "k003-prompt": "How does Proto-Danksharding reduce rollup transaction costs on rollups?",
+  "k003-a-label": "Directly increasing the gas limit per block",
+  "k003-a-explanation": "Proto-Danksharding does not directly increase the gas limit, but does make the storage of rollup data less expensive by making temporary storage available",
+  "k003-b-label": "Splitting up which validators are required to store the data",
+  "k003-b-explanation": "Although full Danksharding is expected to reduce the need for all validators to store all of the data, this is preceded by Proto-Danksharding which forms a less-expensive, temporary storage option for the data produced by rollups.",
+  "k003-c-label": "Significantly increasing hardware requirements for node operators",
+  "k003-c-explanation": "This is generally not considered an acceptable option for scaling Ethereum. Great efforts are taken to minimize hardware requirements for operating a node to keep it as accessible as possible.",
+  "k003-d-label": "Storing its data in cheaper, temporary 'blob' storage",
+  "k003-d-explanation": "Proto-Danksharding introduces a temporary data storage option for rollups to allow them to more cheaply post its results to Mainnet",
+  "k004-prompt": "What is a critical next step to scaling Ethereum with rollups?",
+  "k004-a-label": "Incentivize entities with powerful computers to handle all of the sequencing",
+  "k004-a-explanation": "One of the problems with current rollups is the centralized nature of those running the sequencers (those who decide the inclusion and ordering of transactions within a rollup). The goal is to allow anyone to participate, and not rely on a single group or entity in any way.",
+  "k004-b-label": "Distribute responsibility for running sequencers and provers across more people",
+  "k004-b-explanation": "Rollups were bootstrapped using centralized sequencers (software run by small groups that decides the inclusion and ordering of transactions within a rollup), but this leaves the network prone to censorship. To prevent the possibility of coercion, sanctioning or bribery, decentralizing the sequencer roll so anyone can participate is essential.",
```

```suggestion
  "k004-b-explanation": "Control over a rollup typically starts off centralized, which helps get things started, but leaves the network prone to censorship. Decentralizing the process of including transactions so anyone can participate is essential to prevent the possibility of network compromise.",
```

@konopkja How does this sound? As I mentioned offline, the terms here were pulled from the copy of the page itself. I tried trimming some of the fat and simplifying for the sake of this explanation.

---

This is better. keeps it high level for more ppl to understand and focuses on the problem, nice.